### ISSUE TITLE
Errors when running deb-get list after 0.3.7 update

### ISSUE THREAD

**Issue by bcook128** [wimpysworld/deb-get#781]
# Errors when running deb-get list after 0.3.7 update

Since 0.3.7 update, cannot produce a list of installed/available packages. 
Env DEBGET_TOKEN is set to valid github token. 

Tried running:
```bash
deb-get clean
deb-get update
```
Problem continues.

### Expected behavior

Output list of available packages

### Actual behavior
```bash
1password
agena
  [*] WARNING! Cached file /var/cache/deb-get/android-messages-desktop.json is empty or missing.
android-messages-desktop
  [*] WARNING! Cached file /var/cache/deb-get/antimicrox.json is empty or missing.
antimicrox
anydesk
appimagelauncher                [ installed ]
atom
audio-recorder
azure-cli
  [*] WARNING! Cached file /var/cache/deb-get/azuredatastudio.json is empty or missing.
azuredatastudio
  [*] WARNING! Cached file /var/cache/deb-get/balena-etcher-electron.json is empty or missing.
balena-etcher-electron
bat                             [ installed ]
  [*] WARNING! Cached file /var/cache/deb-get/battery-monitor.json is empty or missing.
battery-monitor
beersmith3
  [*] WARNING! Cached file /var/cache/deb-get/bitwarden.json is empty or missing.
bitwarden
bitwig-studio
blanket
  [*] WARNING! Cached file /var/cache/deb-get/blockbench.json is empty or missing.
blockbench
bluejeans-v2
  [*] WARNING! Cached file /var/cache/deb-get/bottom.json is empty or missing.
bottom
brave-browser
  [*] WARNING! Cached file /var/cache/deb-get/brisqi.json is empty or missing.
brisqi
  [*] WARNING! Cached file /var/cache/deb-get/caprine.json is empty or missing.
caprine
cawbird
chronograf
code                            [ installed ]
  [*] WARNING! Cached file /var/cache/deb-get/codium.json is empty or missing.
codium
  [*] WARNING! Cached file /var/cache/deb-get/com.github.tkashkin.gamehub.json is empty or missing.
com.github.tkashkin.gamehub
copyq
  [*] WARNING! Cached file /var/cache/deb-get/cosign.json is empty or missing.
cosign
  [*] WARNING! Cached file /var/cache/deb-get/croc.json is empty or missing.
croc
cronopete
crossover
cryptomator
dbeaver-ce
deadbeef-static
deb-get                         [ installed ]
  [*] WARNING! Cached file /var/cache/deb-get/deborah.json is empty or missing.
deborah
deltachat-desktop
  [*] WARNING! Cached file /var/cache/deb-get/deskcut.json is empty or missing.
deskcut
discord
docker-ce
docker-desktop
  [*] WARNING! Cached file /var/cache/deb-get/draw.io.json is empty or missing.
draw.io
dropbox
  [*] WARNING! Cached file /var/cache/deb-get/du-dust.json is empty or missing.
du-dust
  [*] WARNING! Cached file /var/cache/deb-get/duf.json is empty or missing.
duf
  [*] WARNING! Cached file /var/cache/deb-get/dustracing2d.json is empty or missing.
dustracing2d
element-desktop
enpass
exodus
expressvpn
  [*] WARNING! Cached file /var/cache/deb-get/fastfetch.json is empty or missing.
fastfetch
  [*] WARNING! Cached file /var/cache/deb-get/fd.json is empty or missing.
fd
  [*] WARNING! Cached file /var/cache/deb-get/ferdium.json is empty or missing.
ferdium
  [*] WARNING! Cached file /var/cache/deb-get/figma-linux.json is empty or missing.
figma-linux
firefox-esr
  [*] WARNING! Cached file /var/cache/deb-get/flameshot.json is empty or missing.
flameshot
foliate
  [*] WARNING! Cached file /var/cache/deb-get/franz.json is empty or missing.
franz
  [*] WARNING! Cached file /var/cache/deb-get/frostwire.json is empty or missing.
frostwire
fsearch
  [*] WARNING! Cached file /var/cache/deb-get/gcm.json is empty or missing.
gcm
gh                              [ installed ]
  [*] WARNING! Cached file /var/cache/deb-get/git-delta.json is empty or missing.
git-delta
  [*] WARNING! Cached file /var/cache/deb-get/github-desktop.json is empty or missing.
github-desktop
gitkraken
gitter
  [*] WARNING! Cached file /var/cache/deb-get/google-chat-electron.json is empty or missing.
google-chat-electron
google-chrome-stable            [ installed ]
google-cloud-cli
google-earth-pro-stable
gpu-viewer
  [*] WARNING! Cached file /var/cache/deb-get/grype.json is empty or missing.
grype
headset
helio-workstation
  [*] WARNING! Cached file /var/cache/deb-get/heroic.json is empty or missing.
heroic
  [*] WARNING! Cached file /var/cache/deb-get/homeassistant-supervised.json is empty or missing.
homeassistant-supervised
  [*] WARNING! Cached file /var/cache/deb-get/hugo.json is empty or missing.
hugo
  [*] WARNING! Cached file /var/cache/deb-get/hyper.json is empty or missing.
hyper
  [*] WARNING! Cached file /var/cache/deb-get/igdm.json is empty or missing.
igdm
indicator-sound-switcher
influxdb
influxdb2
influxdb2-cli
  [*] WARNING! Cached file /var/cache/deb-get/insomnia.json is empty or missing.
insomnia
insync
  [*] WARNING! Cached file /var/cache/deb-get/ipfs-desktop.json is empty or missing.
ipfs-desktop
  [*] WARNING! Cached file /var/cache/deb-get/irccloud-desktop.json is empty or missing.
irccloud-desktop
iriunwebcam
  [*] WARNING! Cached file /var/cache/deb-get/jabref.json is empty or missing.
jabref
jami
jellyfin
  [*] WARNING! Cached file /var/cache/deb-get/jupyterlab-desktop.json is empty or missing.
jupyterlab-desktop
kapacitor
kdiskmark
keepassxc
keybase
kopia-ui
  [*] WARNING! Cached file /var/cache/deb-get/ksnip.json is empty or missing.
ksnip
lens
libreoffice
librewolf
  [*] WARNING! Cached file /var/cache/deb-get/limoji.json is empty or missing.
limoji
lsd                             [ installed ]
  [*] WARNING! Cached file /var/cache/deb-get/ludo.json is empty or missing.
ludo
  [*] WARNING! Cached file /var/cache/deb-get/lutris.json is empty or missing.
lutris
mailspring                      [ installed ]
makedeb
master-pdf-editor-5
  [*] WARNING! Cached file /var/cache/deb-get/mattermost-desktop.json is empty or missing.
mattermost-desktop
media-downloader
mediathekview
  [*] WARNING! Cached file /var/cache/deb-get/mergerfs.json is empty or missing.
mergerfs
  [*] WARNING! Cached file /var/cache/deb-get/micro.json is empty or missing.
micro
microsoft-edge-stable
  [*] WARNING! Cached file /var/cache/deb-get/min.json is empty or missing.
min
  [*] WARNING! Cached file /var/cache/deb-get/minigalaxy.json is empty or missing.
minigalaxy
  [*] WARNING! Cached file /var/cache/deb-get/minikube.json is empty or missing.
minikube
  [*] WARNING! Cached file /var/cache/deb-get/motrix.json is empty or missing.
motrix
  [*] WARNING! Cached file /var/cache/deb-get/mpdevil.json is empty or missing.
mpdevil
  [*] WARNING! Cached file /var/cache/deb-get/ms-office-electron.json is empty or missing.
ms-office-electron
  [*] WARNING! Cached file /var/cache/deb-get/mullvad-vpn.json is empty or missing.
mullvad-vpn
nala                            [ installed ]
neo4j
neovim                          [ installed ]
nextcloud-desktop
nomad
nordvpn
  [*] WARNING! Cached file /var/cache/deb-get/notable.json is empty or missing.
notable
  [*] WARNING! Cached file /var/cache/deb-get/obs-cli.json is empty or missing.
obs-cli
obsidian                        [ installed ]
obs-studio
ocenaudio
onedriver
  [*] WARNING! Cached file /var/cache/deb-get/onlyoffice-desktopeditors.json is empty or missing.
onlyoffice-desktopeditors
  [*] WARNING! Cached file /var/cache/deb-get/openaudible.json is empty or missing.
openaudible
openrazer-meta
opera-stable
  [*] WARNING! Cached file /var/cache/deb-get/os-agent.json is empty or missing.
os-agent
  [*] WARNING! Cached file /var/cache/deb-get/p3x-onenote.json is empty or missing.
p3x-onenote
pandoc                          [ installed ]
papirus-icon-theme
  [*] WARNING! Cached file /var/cache/deb-get/peazip.json is empty or missing.
peazip
plexmediaserver
  [*] WARNING! Cached file /var/cache/deb-get/plow.json is empty or missing.
plow
polychromatic
portmaster
  [*] WARNING! Cached file /var/cache/deb-get/powershell.json is empty or missing.
powershell
  [*] WARNING! Cached file /var/cache/deb-get/protonmail-bridge.json is empty or missing.
protonmail-bridge
protonvpn
  [*] WARNING! Cached file /var/cache/deb-get/pulse-sms.json is empty or missing.
pulse-sms
qownnotes
  [*] WARNING! Cached file /var/cache/deb-get/quarto.json is empty or missing.
quarto
quickemu
quickgui
  [*] WARNING! Cached file /var/cache/deb-get/quickobs.json is empty or missing.
quickobs
  [*] WARNING! Cached file /var/cache/deb-get/rambox.json is empty or missing.
rambox
  [*] WARNING! Cached file /var/cache/deb-get/rclone.json is empty or missing.
rclone
resilio-sync
retroarch
  [*] WARNING! Cached file /var/cache/deb-get/rocketchat.json is empty or missing.
rocketchat
  [*] WARNING! Cached file /var/cache/deb-get/rpi-imager.json is empty or missing.
rpi-imager
rstudio
rstudio-server
  [*] WARNING! Cached file /var/cache/deb-get/rustdesk.json is empty or missing.
rustdesk
sejda-desktop
shutter-encoder
signal-desktop
  [*] WARNING! Cached file /var/cache/deb-get/simplenote.json is empty or missing.
simplenote
skypeforlinux
slack-desktop
  [*] WARNING! Cached file /var/cache/deb-get/sleek.json is empty or missing.
sleek
smartgit                        [ installed ]
softmaker-office-2021
  [*] WARNING! Cached file /var/cache/deb-get/soundux.json is empty or missing.
soundux
spotify-client
  [*] WARNING! Cached file /var/cache/deb-get/standard-notes.json is empty or missing.
standard-notes
strawberry
sublime-merge
sublime-text
  [*] WARNING! Cached file /var/cache/deb-get/sunshine.json is empty or missing.
sunshine
surfshark
  [*] WARNING! Cached file /var/cache/deb-get/syft.json is empty or missing.
syft
syncthing
  [*] WARNING! Cached file /var/cache/deb-get/system-monitoring-center.json is empty or missing.
system-monitoring-center
  [*] WARNING! Cached file /var/cache/deb-get/tabby-terminal.json is empty or missing.
tabby-terminal
tailscale
tarsnap
teams
  [*] WARNING! Cached file /var/cache/deb-get/teams-for-linux.json is empty or missing.
teams-for-linux
teamviewer
telegraf
terraform
texworks
  [*] WARNING! Cached file /var/cache/deb-get/tidal-hifi.json is empty or missing.
tidal-hifi
tixati
tonelib-bassdrive
tonelib-gfx
tonelib-jam
tonelib-metal
tonelib-noisereducer
tonelib-tubewarmth
tonelib-zoom
  [*] WARNING! Cached file /var/cache/deb-get/tribler.json is empty or missing.
tribler
  [*] WARNING! Cached file /var/cache/deb-get/trivy.json is empty or missing.
trivy
typora
ubuntu-make
ulauncher
urserver
usbimager
vagrant
veracrypt
virtualbox-6.1
virtualbox-7.0
vivaldi-stable
vuescan
wavebox
waydroid
webex
weechat
  [*] WARNING! Cached file /var/cache/deb-get/weylus.json is empty or missing.
weylus
  [*] WARNING! Cached file /var/cache/deb-get/whalebird.json is empty or missing.
whalebird
  [*] WARNING! Cached file /var/cache/deb-get/whatsapp-for-linux.json is empty or missing.
whatsapp-for-linux
winehq-devel
winehq-stable
winehq-staging
wire-desktop
  [*] WARNING! Cached file /var/cache/deb-get/wraith-master.json is empty or missing.
wraith-master
xemu
  [*] WARNING! Cached file /var/cache/deb-get/youtube-music.json is empty or missing.
youtube-music
yq
zenith                          [ installed ]
  [*] WARNING! Cached file /var/cache/deb-get/zettlr.json is empty or missing.
zettlr
zoom
zotero
```

### Steps to reproduce

Run command:  deb-get list

#### /etc/os-release

```bash
PRETTY_NAME="Ubuntu 22.04.2 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.2 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

#### /etc/lsb-release

```bash
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.2 LTS"

```

#### deb-get version

```bash
0.3.7
```

### deb-get update

```bash
  [+] Updating /etc/deb-get/01-main
/etc/deb-get/01-main.repo                              100%[===========================================================================================================================>]   2.70K  --.-KB/s    in 0s      
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  102k  100  102k    0     0  89259      0  0:00:01  0:00:01 --:--:-- 89259
/home/brad
Hit:1 http://deb.volian.org/volian scar InRelease
Hit:2 http://us.archive.ubuntu.com/ubuntu jammy InRelease
Hit:3 http://us.archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:4 https://dl.google.com/linux/chrome/deb stable InRelease
Hit:5 http://us.archive.ubuntu.com/ubuntu jammy-backports InRelease
Hit:6 https://packages.microsoft.com/repos/vscode stable InRelease
Hit:7 http://security.ubuntu.com/ubuntu jammy-security InRelease
Hit:8 https://ppa.launchpadcontent.net/kubuntu-ppa/backports-extra/ubuntu jammy InRelease
Hit:9 https://ppa.launchpadcontent.net/kubuntu-ppa/backports/ubuntu jammy InRelease
Hit:10 https://ppa.launchpadcontent.net/ultradvorka/ppa/ubuntu jammy InRelease
Reading package lists...

```

**Comment by philclifford** [comment]
It is possible github has been having issues today.  I just checked on jammy with the 0.3.7 release and got a clean update (although one package got an error on the first fetch, but since it is installed a second fetch (which normally would not happen because it would be cached and fresh) succeeded fine.  However you shouldn't see those errors in the `list`
Can you please check if you have  `/var/cache/deb-get/updating_supported.lock` (and if you do, `sudo rm /var/cache/deb-get/updating_supported.lock` ) 
and retry a `deb-get update` ?

**Comment by philclifford** [comment]
Also, might you be hitting the problem fixed in #774 ?
